content/influxdb/cloud/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,20 @@ menu:
     name: influx CLI 
 ---
 
+## v2.1.1 [unreleased]
```

Comment:
Everything other than this file seems to be unrelated to the CLI release?

Reply:
Hi @danxmoran! Yes I believe I made a mistake! This is strange though because I made this branch immediately before adding edits to the release notes doc and then sent it out immediately after I was done with the work... and when I was staging changes VS code showed me that only that one doc was affected.... But yes, everything in the reference notes doc is only thing related to this PR! Sorry about that!